contrib/seccomp/kernel_linux.go

```diff
@@ -0,0 +1,78 @@
+//go:build linux
```

Comment:
I'm feeling this should be a separate package like `pkg/kernelversion` or `contrib/seccomp/kernelversion`